config/overlays/openshift/controller_manager_config.yaml

```
@@ -30,6 +30,7 @@ featureGates:
     openshiftRoute: true
     servingCertsService: true
   prometheusOperator: true
+  grafanaOperator: true
```

```suggestion
  grafanaOperator: false
```

The operator is not installed by default on OpenShift.

---

Will change it.